ci/cloudbuild/cache.sh

```diff
@@ -39,6 +39,11 @@ function print_usage() {
   sed -n '17,/^$/s/^# \?//p' "${PROGRAM_PATH}"
 }
 
+io::log "Running:"
+printf "env -C '%s' %s" "$(pwd)" "$0"
+printf " %q" "$@"
+printf "\n"
```

Comment:
nit: can one say `printf " %q\n" "$@"` ?

Reply:
No. That would run `" %q\n` for each arg, printing each one, escaped, on its own line.